features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/Rum.kt

```diff
@@ -73,6 +73,14 @@ object Rum {
 
         sdkCore.registerFeature(rumFeature)
 
+        sdkCore.getFeature(rumFeature.name)?.dataStore?.let {
+            val anonymousIdentifierManager = AnonymousIdentifierManager(
```

Comment:
> [!NOTE]
> Not sure that we need to create an instance of stateless class just to call one method... Maybe some static function will be better here?

Reply:
Definitely took the shortcut here. I'll look into it 🙌